lib/core/src/display/shape/text/glyph/system.rs

```diff
@@ -186,43 +192,36 @@ impl GlyphSystem {
         let instance_id     = sprite.instance_id();
         let color_attr      = self.color.at(instance_id);
         let msdf_index_attr = self.glyph_msdf_index.at(instance_id);
-        let font_id         = self.font_id;
+        let font            = self.font.clone();
```

Comment:
if `font` implementing `clone_ref`? If this is a cheap operation we should tell about it here.